react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+export default function Custom() {
+  const [begin, isClicked] = useState(0);
+
+  return (
```

Comment:
It would be better to calculate the class name above and have a simple JSX component that just passes the class name to `className`. You will notice in your code that you repeat the JSX component over and over again. Precalculating the class name avoids this.